### ISSUE TITLE
Не загружаются аватарки некоторых (не всех) пользователей, баннера,  некоторые эмодзи

### ISSUE THREAD

**Issue by MexrterPs** [Flowseal/zapret-discord-youtube#607]
Решил поменять аватарку и баннер, итог - теперь у меня прозрачная ава и чёрный баннер и это никак не фикситься, уже находил тут эту проблему, но решения в комментариях не помогли (не значительные) винду менять не буду, скачать бету в принципе могу, ещё добавил в лист cloudflare-ech.com. Ну и у других пользователей тоже прозрачные аватарки и чёрные баннеры ну или белые. Помогите
![изображение_2024-11-06_212815148](https://github.com/user-attachments/assets/410d2c6c-539b-4aca-9fc9-9c71bb474d9d)
![изображение_2024-11-06_212851250](https://github.com/user-attachments/assets/9f701eea-a4b7-4da5-acdf-9b970f0a1752)

**Comment by ch3buxs** [comment]
такая же лажа

**Comment by NoNameThing** [comment]
--hostlist="list-general.txt" --dpi-desync=fake,split 
Поменять на 
 --ipset="ipset-discord.txt" --dpi-desync=disorder
 
Редактировать .bat, который ты запускаешь 
Пкм 
Редактировать в блокноте

**Comment by MexrterPs** [comment]
> --hostlist="list-general.txt" --dpi-desync=fake,split Поменять на --ipset="ipset-discord.txt" --dpi-desync=disorder
> 
> Редактировать .bat, который ты запускаешь Пкм Редактировать в блокноте

у меня сервис, я в ручную не запсукаю, но спасибо, правда поможет ли мне это с сервисом?

**Comment by nyahnem** [comment]
Мне помогло добавить cloudflare-ech.com в лист, перезапуск сервиса все починил после, спасибо!

**Comment by DenisShmyrev** [comment]
> Мне помогло добавить cloudflare-ech.com в лист, перезапуск сервиса все починил после, спасибо!

Возможно некто снова что-то заблокал, поэтому добавление этого в список вернуло их обратно

**Comment by TemaTime** [comment]
> --hostlist="list-general.txt" --dpi-desync=fake,split Поменять на --ipset="ipset-discord.txt" --dpi-desync=disorder
> 
> Редактировать .bat, который ты запускаешь Пкм Редактировать в блокноте

Строчки указанной вами нету, что делать в таком случае?
![image](https://github.com/user-attachments/assets/af22b762-2b92-4838-babe-34b1cbacf479)

**Comment by MexrterPs** [comment]
Да, это работает, нужно добавить cloudflare-ech.com и перезагрузить сервис, если он у вас установлен, перезагрузить таким образом:
1.удалить сервис (используя пкм - открыть от имени администратора)
2.установить сервис (используя пкм - открыть от имени администратора)
3.перезагрузить пк
Готово!

**Comment by MexrterPs** [comment]
Но проблема не исправна до конца, например если зайти к кому то в профиль или к себе, то проблема останется, кроме того
![image](https://github.com/user-attachments/assets/259a2df5-5d04-4169-b958-51072d7ce865)
![image](https://github.com/user-attachments/assets/55e23b49-4ebc-44e2-8e3a-6f3a397a3a76)
![image](https://github.com/user-attachments/assets/8436b988-00c3-4a8c-8fa7-2a6d6af872de)

**Comment by codebykoi** [comment]
> --hostlist="list-general.txt" --dpi-desync=fake,split Поменять на --ipset="ipset-discord.txt" --dpi-desync=disorder
> 
> Редактировать .bat, который ты запускаешь Пкм Редактировать в блокноте

Проблема только в том что в связке с YouTube работать не будет.

**Comment by DanzoGit** [comment]
Мне запрет давно для ютуба не помогает. Только юбуст, либо варп.

**Comment by MexrterPs** [comment]
> Мне запрет давно для ютуба не помогает. Только юбуст, либо варп.

странно, у меня всё отлично работает

**Comment by MexrterPs** [comment]
Неожиданно, сейчас всё отлично, я вижу все аватарки, даже если захожу в профиль, вижу все эмодзи, если это исправили разработчики, спасибо!

**Comment by MexrterPs** [comment]
Неожиданно, но я сейчас зашёл и я вижу аватарки, баннеры и эмодзи, правда не все.., проблемы частично больше нет, если это дела рук разработчиков, спасибо!

**Comment by TheDoukiDrift** [comment]
> --hostlist="list-general.txt" --dpi-desync=fake,split Поменять на --ipset="ipset-discord.txt" --dpi-desync=disorder
> 
> Редактировать .bat, который ты запускаешь Пкм Редактировать в блокноте

У меня перестал из за этого работать Ютуб, есть предложения?

**Comment by MexrterPs** [comment]
> > --hostlist="list-general.txt" --dpi-desync=fake,split Поменять на --ipset="ipset-discord.txt" --dpi-desync=disorder
> > Редактировать .bat, который ты запускаешь Пкм Редактировать в блокноте
> 
> У меня перестал из за этого работать Ютуб, есть предложения?

Не надо этого делать, можно в лист добавить cloudflare-ech.com, как я говорил ранее и сейчас вроде фиксят всё, ну у меня начинает работать, с задержкой конечно

**Comment by TheDoukiDrift** [comment]
> > > --hostlist="list-general.txt" --dpi-desync=fake,split Поменять на --ipset="ipset-discord.txt" --dpi-desync=disorder
> > > Редактировать .bat, который ты запускаешь Пкм Редактировать в блокноте
> > 
> > 
> > У меня перестал из за этого работать Ютуб, есть предложения?
> 
> Не надо этого делать, можно в лист добавить cloudflare-ech.com, как я говорил ранее и сейчас вроде фиксят всё, ну у меня начинает работать, с задержкой конечно

У меня так и было, но аватарки не грузились

**Comment by TheDoukiDrift** [comment]
Терь в принципе ютуб не работает

**Comment by TheDoukiDrift** [comment]
А не, ожил

**Comment by MexrterPs** [comment]
> Терь в принципе ютуб не работает

переустанови zapret или сервис, если ты им пользуешься

**Comment by Kovi76** [comment]
> Мне помогло добавить cloudflare-ech.com в лист, перезапуск сервиса все починил после, спасибо!

Всем Привет! Да этот способ действительно помог, СПАСИБО!!!

**Comment by Anozer41** [comment]
> --hostlist="list-general.txt" --dpi-desync=fake,split Поменять на --ipset="ipset-discord.txt" --dpi-desync=disorder
> 
> Редактировать .bat, который ты запускаешь Пкм Редактировать в блокноте

Помогло, спасибо. Для тех кто ставит сервисом - делаем указанную замену в general.bat любым текстовым редактором. Далее переустанавливаем сервис через service_install.bat, вводим 10 (как раз где general.bat). 
![image](https://github.com/user-attachments/assets/5f5af841-3f5f-4ede-b516-c2c029e3a92e)
![image](https://github.com/user-attachments/assets/fd6cbec2-5874-44ff-a8ee-4934bad6df5e)



Надеюсь спасу кому-нибудь лишние 10-15мин)
UPD: хотя мог бы просто файл скинуть... можно просто на .bat поменять и заменить
[general.txt](https://github.com/user-attachments/files/18364627/general.txt)

**Comment by emirskiy95** [comment]
@Anozer41 Спасибо за файл, помогло!
